spaces/models.py

```diff
@@ -0,0 +1,146 @@
+from django.db    import models
+from users.models import Host
+
+class Space(models.Model):
+    host         = models.ForeignKey('users.Host', on_delete=models.CASCADE)
+    name         = models.CharField(max_length=100)
+    price        = models.DecimalField(max_digits=18 ,decimal_places=2)
+    description  = models.CharField(max_length=3000)
+    place_type   = models.ForeignKey('spaces.PlaceType', on_delete=models.CASCADE)
+    sub_property = models.ForeignKey('spaces.SubProperty', on_delete=models.CASCADE)
+    rating       = models.DecimalField(max_digits=3, decimal_places=2, null=True)
+    pub_date     = models.DateField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'spaces'
+
+
+class PlaceType(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'place_types'
+
+
+class Property(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'properties'
+
+
+class SubProperty(models.Model):
+    space_property = models.ForeignKey('spaces.Property', on_delete=models.CASCADE) 
+    name           = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'sub_properties'
+
+
+class Option(models.Model):
+    space    = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    name     = models.CharField(max_length=45)
+    icon_url = models.URLField(max_length=2000)
+
+    class Mata:
+        db_table = 'options'
+
+
+class SpaceOption(models.Model):
+    space  = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    option = models.ForeignKey('spaces.Option', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'space_options'
+
+
+class Location(models.Model):
+    space          = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    country        = models.ForeignKey('users.Country', on_delete=models.CASCADE)
+    region         = models.CharField(max_length=45)
+    city           = models.CharField(max_length=20)
+    address        = models.CharField(max_length=150)
+    address_detail = models.CharField(max_length=45)
+    letitude       = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+    longitude      = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+
+    class Meta:
+        db_table = 'locations'
+
+
+class Image(models.Model):
+    space     = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    image_url = models.URLField(max_length=2000)
```

Comment:
queryset을 다룰 때, 가독성을 위해서 아래와 같이 수정해주세요~
`Image.image_url` vs `image.url`
```suggestion
    url = models.URLField(max_length=2000)
```

Reply:
네!!